### ISSUE TITLE
Pretrained checkpoints

### ISSUE THREAD

**Issue by qing0518** [zhengqili/UprightNet#4]
Excuse me, you have provided a link of pretrained checkpoints in the Readme, but there is no checkpoint.zip documentation in it. Could you please provide the pretrained checkpoints?

**Comment by GotoIlsan** [comment]
Same issue

**Comment by roselaird** [comment]
same issue, did anyone find a solution?

**Comment by snavely** [comment]
Yes, the links seemed to have rotted. I've located what I think are the original checkpoints, reuploaded them, and updated the link on the GitHub page.

https://github.com/zhengqili/UprightNet
